client/src/domain/event/EventPageContainer.tsx

```diff
@@ -26,6 +28,7 @@ const EventPageContainer: React.FC<
               <EventHero eventData={eventData} />
               <Container>
                 <EventContent eventData={eventData} />
+                {isClient && <SimilarEvents eventData={eventData} />}
```

Comment:
Any reason for rendering this only in client?

Reply:
Fetching list of event takes over 1.5 seconds so it makes initial load quite slow